apps/freeablo/faworld/faction.cpp

```diff
@@ -0,0 +1,21 @@
+#include "faction.h"
+
+#include "actor.h"
+
+namespace FAWorld
+{
+
+    bool Faction::canAttack(Faction* other)
+    {
+        // Attacker -> Attackee
+        static bool ruleTable[FactionType::ENUM_END][FactionType::ENUM_END] =
```

Comment:
How about we just have every faction attack other factions for now? Later on we could add in a system that allows dynamically adding alliances, if we need to.